package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "openai",
+  "name": "@get-wrecked/openai",
   "version": "4.29.0",
   "description": "The official TypeScript library for the OpenAI API",
   "author": "OpenAI <support@openai.com>",
   "types": "dist/index.d.ts",
   "main": "dist/index.js",
   "type": "commonjs",
-  "repository": "github:openai/openai-node",
+  "repository": "github:get-wrecked/openai-node",
```

Comment:
These changes are, I hope, unintentionally left in from development

Reply:
yes. Sorry about that. All fixed now. 
Thanks for the quick response.